tests/test_health_gating.matrix

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env matrix
+"tests":
+- "name": Verify gating
+  "description": >
+    This test should fail! We're verifying that our gating on health works.
```

Comment:
@johnsca Agree. We neeed a better way of breaking stuff. I think that we should do that as a separate ticket, though, as this works in the context of our deadline.